models/public/yolo-v3-tf/keras_to_tensorflow.py.patch

```diff
@@ -1,12 +0,0 @@
-diff --git a/keras_to_tensorflow.py b/keras_to_tensorflow.py
```

Comment:
This patch is mentioned in the model documentation. Is that outdated too?

Reply:
Yes, thanks